### PR TITLE
fixing workflow tests

### LINKED ISSUE #33: fix code to satisfy github action workflow
the automated github action workflow in #32 will likely fail. Consider fixes to make all checks pass and have it become a standard for any future prs. 

Paging @wkchuang to help me with this.

### PR DESCRIPTION
**Pull request by ngam**
Fixes #33

### CODE REVIEW

**Comment by ngam** [comment]
@wkchuang these fixes made it all green, don't merge yet, but let me know how you feel about the edits.

File: aerosol_dynamics/particle_test.py
Comment (wkchuang) [inline aerosol_dynamics/particle_test.py:55]: We check units using the decorators in the particle file, so there is no need to test here.
If you feed in the wrong units, it will throw an error at the decorator.
`@u.wraps(u.m, [None])` will only output in meters
Knudsen number is dimensionless, so there's no need to check for units

File: aerosol_dynamics/particle_test.py
Comment (wkchuang) [inline aerosol_dynamics/particle_test.py:76]: That's fine, we can pare it back with sigfigs. These numbers were taken directly from what the code output. I was ensuring that the numbers were not changing as the file was evolving.

File: .github/workflows/python-package.yml
Comment (wkchuang) [inline .github/workflows/python-package.yml:46]: How do I test whether this works?

File: aerosol_dynamics/particle.py
Comment (wkchuang) [inline aerosol_dynamics/particle.py:150]: I'm for separating this one into two or more statements. 

One consideration to keep in mind is between "easier to debug" and "uses fewer resources by not having to store values".

File: .github/workflows/python-package.yml
Comment (ngam) [reply .github/workflows/python-package.yml:46]: It passes the github workflows (on the web)

File: aerosol_dynamics/particle.py
Comment (ngam) [reply aerosol_dynamics/particle.py:150]: That's a good point!

File: aerosol_dynamics/particle_test.py
Comment (ngam) [reply aerosol_dynamics/particle_test.py:76]: Aaaahhhh! Now I understand where these raw numbers come from. Okay, let me think more

File: aerosol_dynamics/particle_test.py
Comment (ngam) [reply aerosol_dynamics/particle_test.py:55]: I see, thanks for explaining. We will want to refine these tests as we go on, but I now understand how you're doing it more fully.

File: aerosol_dynamics/particle.py
Comment (ngam) [reply aerosol_dynamics/particle.py:150]: I added that as another TODO item. Will revist.

**Comment by ngam** [comment]
@wkchuang, this can go, once you approve, please squash and merge or just merge. Thanks :)

I will open two more PRs for the remaining stuff: 1) webpage and 2) more documentation of the mechanics of calculation coagulation (more important when we have more particles anyway)

**Review by wkchuang — APPROVED** [review]

**Pull request merged**
